mobile/src/bundles/talent/helpers/manage-projects-links.ts

```diff
@@ -0,0 +1,11 @@
+type UrlObject = { url: string };
+
+const stringsToUrlObjects = (links: string[] | null): UrlObject[] => {
+    return links?.map((url) => ({ url })) ?? [];
+};
+
+const urlObjectsToStrings = (links: UrlObject[] | null): string[] => {
+    return links?.map(({ url }) => url) ?? [];
+};
```

Comment:
why do we need this?

Reply:
because frontend links should be processed as [{url: ''}, {url: ''}, ...]
But in the database they are stored with the type string[]. That's why we at daily talked about converting them to one or the other way.